operations/finance.md

```diff
@@ -599,47 +592,51 @@ Updates specified accounting items. Allows to change to which account or bill th
 
 ```javascript
 {
-    "AccountingItems": [
+    "OrderItems": [
         {
-            "Id": "ee425738-8112-446f-b764-abd7007ba880",
-            "CustomerId": "a5786a7b-a388-43cc-a838-abd7007b5ff7",
-            "OrderId": "5cb9dc38-642d-412a-a7eb-abd7007b9ecb",
-            "ServiceId": "bc69c610-f0f8-4645-8bb3-ab3a00c97c1e",
-            "ProductId": null,
-            "BillId": "30b4b0c2-5e9c-4247-91d3-abd8005e2a0a",
-            "InvoiceId": null,
-            "AccountingCategoryId": "13cd6f3a-be61-4ce3-9a19-ab3a00b98f49",
-            "CreditCardId": null,
-            "Type": "ProductRevenue",
-            "SubType": "Surcharge",
-            "Name": "Service / Product",
-            "Notes": null,
-            "ConsumptionUtc": "2020-06-11T07:29:00Z",
-            "ClosedUtc": null,
-            "State": "Open",
+            "Id": "6c2897de-620a-4f48-af1e-ada8004202bd",
+            "AccountId": "182a56ee-037d-4da5-b6f8-ada8006e7d5c",
+            "OrderId": "f9090129-fb49-46d2-9dc5-ad9d015d43b9",
+            "BillId": "9e3791dc-95c7-439a-aa8a-ada8007de0ca",
+            "AccountingCategoryId": "d250149e-a29d-4c70-b607-a1759faf7320",
             "Amount": {
-                "Value": 52.8,
-                "Net": 48.0,
-                "Tax": 4.8,
-                "TaxRate": 0.1,
-                "Currency": "USD",
-                "NetValue": 48.0,
-                "GrossValue": 52.8,
+                "Currency": "GBP",
+                "NetValue": 95.24,
+                "GrossValue": 100.00,
                 "TaxValues": [
                     {
-                        "Code": "US-DC-I",
-                        "Value": 4.8
+                        "Code": "UK-2020-R",
+                        "Value": 4.76
                     }
-                ]
+                ],
+                "Breakdown": {
+                    "Items": [
+                        {
+                            "TaxRateCode": "UK-2020-R",
+                            "NetValue": 95.24,
+                            "TaxValue": 4.76
+                        }
+                    ]
+                }
+            },
+            "RevenueType": "Additional",
+            "ConsumedUtc": "2021-09-19T04:00:20Z",
+            "ClosedUtc": null,
+            "AccountingState": "Open",
+            "Data": {
+                "Discriminator": "CancellationFee",
+                "Value": null
             }
         }
-    ]
+    ],
+    "PaymentItems": [],
 }
 ```
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `AccountingItems` | array of [Accounting item](#accounting-item) | optional | The updated accounting items. |
+| `OrderItems` | array of [Order item](#order-item) | optional | Updated order items (consumed items such as nights or products). |
+| `PaymentItems` | array of [Payment item](#payment-item) | optional | Updated payment items (such as cash, credit card payments or invoices). |
```

Comment:
nit: For clarity, I wouldn't mention invoices here. If we really want to, let's go with invoice payments, but if you shorten it to just invoice, it might be ambiguous with bills of type invoice. So for the clarity of whole documentation, it might be best to refer to those two things as invoice payments and invoice bills (which is something I've already seen in the docs).